enterprise_access/apps/enterprise_groups/management/commands/groups_reminder_emails.py

```diff
@@ -49,7 +49,7 @@ def handle(self, *args, **options):
                 policy_group_association.subsidy_access_policy.subsidy_expiration_datetime
             )
             catalog_uuid = policy_group_association.subsidy_access_policy.catalog_uuid
-            catalog_count = enterprise_catalog_client.catalog_content_metadata(
+            catalog_count = enterprise_catalog_client.get_content_metadata_count(
                 catalog_uuid
             )['count']
```

Comment:
Does this work?  `get_content_metadata_count` looks to be returning a bare value, not an object.